resources/dicts/events/death/beach/apprentice.json

```diff
@@ -1,3 +1,225 @@
+        "history_text": {
+                     "reg_death": "m_c died due to dehydration and heat exhaustion."
+                 },
+        "cat_trait": null,
+         "cat_skill": null,
+         "other_cat_trait": null,        
+         "other_cat_skill": null 
+     },
+     {
+        "camp": "any",
+        "tags": [
+         "Newleaf", 
+         "Greenleaf", 
+         "Leaf-fall",
+         "other_cat",
+         "other_cat_parent"
+        ],
+        "death_text": "r_c is frantic with worry; m_c, {PRONOUN/r_c/poss} child, hadn't been seen around camp, no matter who {PRONOUN/r_c/subject} asked and where {PRONOUN/r_c/subject} searched. It's not until r_c stumbles across m_c's body, withered under the sun and being feasted on by black and orange butterflies, with a despairing sound, that {PRONOUN/r_c/subject} {VERB/r_c/realize/realizes} {PRONOUN/r_c/poss} greatest fears have come true.",
+        "history_text": {
+                     "reg_death": "m_c shriveled up under the sun because of heat exhaustion and dehydration."
+                 },
+        "cat_trait": null,
+        "cat_skill": null,
+        "other_cat_trait": null,        
+        "other_cat_skill": null 
+    },
+    {
+        "camp": "any",
+        "tags": [
+            "Newleaf", 
+            "Greenleaf", 
+            "Leaf-fall"
+        ],
+        "death_text": "m_c couldn't speak, yell for help, or beg StarClan to give {PRONOUN/m_c/subject} a sign they were still with {PRONOUN/m_c/object}, {PRONOUN/m_c/poss}  tongue having swelled up inside {PRONOUN/m_c/poss} dried mouth. As m_c collapses, feeling {PRONOUN/m_c/poss} body internally shut down because of dehydration and heat exhaustion, {PRONOUN/m_c/subject} pray that {PRONOUN/m_c/poss} clanmates locate m_c's body before {PRONOUN/m_c/subject} become unrecognizable under the sun.",
+        "history_text": {
+                        "reg_death": "m_c was claimed by dehydration and heat exhaustion."
+                        },
+        "cat_trait": null,
+        "cat_skill": null,
+        "other_cat_trait": null,        
+        "other_cat_skill": null
+     },
+     {
+        "camp": "any",
+        "tags": [
+            "Newleaf", 
+            "Greenleaf", 
+            "Leaf-fall",
+            "other_cat",
+            "other_cat_mentor"
+        ],
+        "death_text": "m_c knew {PRONOUN/m_c/subject} made a mistake, as {PRONOUN/m_c/subject} felt {PRONOUN/m_c/poss} last bits of strength slip away from {PRONOUN/m_c/object} and {PRONOUN/m_c/poss} heart stopped beating. {PRONOUN/m_c/poss/CAP} body only discovered days later, r_c wailing at the shriveled body of {PRONOUN/r_c/poss} apprentice, who succumbed to heat and dehydration, covered in beetles.",
```

Comment:
This one borderline, but it may be best to avoid anything that implies insects are eating the body.